docs/content/preview/explore/fault-tolerance/transaction-availability.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Availability of Transactions
+headerTitle:  Availability of Transactions
+linkTitle: Availability of Transactions
+description: Simulate fault tolerance and resilience of transactions in a local YugabyteDB database universe.
+headcontent: Highly available and fault tolerant transactions
+aliases:
+  - /explore/fault-tolerance/transactions/
+  - /preview/explore/fault-tolerance/transactions/
+  - /preview/explore/cloud-native/fault-tolerance/transactions/
+  - /preview/explore/postgresql/fault-tolerance/transactions/
+  - /preview/explore/fault-tolerance-macos/transactions/
+menu:
+  preview:
+    identifier: transaction-availability-local
+    parent: fault-tolerance-1-macos
+    weight: 215
+type: docs
+---
+
+[Transactions](../../../architecture/transactions/distributed-txns/) are critical to many applications and need to work through different failure scenarios.  YugabyteDB provides [high availability](../../../architecture/core-functions/high-availability/) (HA) of transactions by replicating the uncommitted values, a.k.a [provisional records](../../../architecture/transactions/distributed-txns/#provisional-records) across the [fault domains](../../../architecture/docdb-replication/replication/#fault-domains). The following examples demonstrate how YugabyteDB transactions survive common failure scenarios that could happen when a transaction is being processed. Some of the scenarios are:
+- The node that has received the provisional write fails (Handled by YugabyteDB)
+- The node that is about to receive the provisional write fails (Handled by YugabyteDB)
+- The transaction co-ordinator fails (Retry by client)
+
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../macos/transaction-availability" class="nav-link active">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Local
+    </a>
+  </li>
+</ul>
+
+{{% explore-setup-single %}}
+
+Follow the [setup instructions](../../#set-up-yugabytedb-universe) to start a single region three-node universe. To verify that the cluster is running correctly, navigate to the application UI at <http://localhost:7000/> to view the status of the nodes in the cluster.
+
+Create the Table.
+```sql
+CREATE TABLE txndemo (
+    k1 int,
+    k2 int,
+    PRIMARY KEY(k1)
+);
+```
+Insert some sample data and determine hashcode of the primary keys.
+
+```sql
+INSERT INTO txndemo SELECT id,10 FROM generate_series(1,5) AS id;
+```
+
+### Identifying the Key location
+For the purpose of the examples, you would need to identify which node holds a specific row, so that you
+can kill the correct node to see what is happening. For this one needs to get the hash code of the primary key using the `yb_hash_code(id)` function and co-relate with the `yb-admin` output to figure out where that 
+key is located. For the examples we will focus on the row with `k1=1`.
```

Comment:
why is this separated across lines?

Reply:
Probably an erroneous <Enter> . But rendering is correct. Will fix it.